src/devices/Ssd1351/Command/Locking.cs

```diff
@@ -0,0 +1,104 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.Ssd1351.Command
+{
+    public class Lock : ICommand
+    {
+        /// <summary>
+        /// This command is used to lock the OLED driver IC from accepting any command except itself.
+        /// </summary>
+        public Lock()
```

Comment:
Seems to me that lock and unlock could be one command with a bool flag (similarly to how the hardware works)